src/fsharp/FSharp.Build/Microsoft.FSharp.Targets

```diff
@@ -89,24 +89,28 @@ this file.
 
         <CreateFSharpManifestResourceName
              Condition="'@(ResxWithNoCulture)' != '' AND '$(UsingXBuild)' == 'true'"
+             UseStandardResourceNames="$(UseStandardResourceNames)"
```

Comment:
@KevinRansom Does anything need to be done for .NET SDK targets here?

Reply:
@KevinRansom Is this fix in the latest .NET SDK or will it be in a future release? I cannot get it to work on my project, which looks like this:

```xml
<PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.0</TargetFramework>
    <RootNamespace>Generators</RootNamespace>
    <UseStandardResourceNames>true</UseStandardResourceNames>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="Program.fs" />
  </ItemGroup>

  <ItemGroup>
    <EmbeddedResource Include="Templates\*.liquid" />
  </ItemGroup>
```

If I use `ildasm` to look at the manifest, the resource doesn't have the prepended namespace:

```.mresource public _TestClass.liquid```